mailpoet/tests/integration/Newsletter/Scheduler/PostNotificationTest.php

```diff
@@ -63,22 +71,26 @@ public function testItCreatesPostNotificationSendingTask() {
     ]);
 
     // new queue record should be created
-    $queue = $this->postNotificationScheduler->createPostNotificationSendingTask($newsletter);
-    assert($queue instanceof SendingTask);
-    expect(SendingQueue::where('newsletter_id', $newsletter->getId())->findMany())->count(1);
-    expect($queue->newsletterId)->equals($newsletter->getId());
-    expect($queue->status)->equals(SendingQueue::STATUS_SCHEDULED);
+    $sendingQueue = $this->postNotificationScheduler->createPostNotificationSendingTask($newsletter);
+    assert($sendingQueue instanceof SendingQueueEntity);
```

Comment:
We want to remove all asserts from our codebase. There already exists [a ticket](https://mailpoet.atlassian.net/browse/MAILPOET-4258) for this.
If I'm not wrong, you can use `$this->assertInstanceOf()` to avoid errors from PHPStan.

You don't have to change the old code, but we shouldn't add more of those cases.